packages/dashboard/src/components/sidePanel/sections/legendSettingSection/index.spec.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import LegendSettings from './index';
+import { DefaultDashboardMessages } from '../../../../messages';
+import { configureDashboardStore } from '../../../../store';
+import { Widget } from '../../../../types';
+import { MOCK_KPI_WIDGET } from '../../../../../testing/mocks';
+import { DashboardState } from '../../../../store/state';
+import { render } from '@testing-library/react';
+
+const widget: Widget = {
+  ...MOCK_KPI_WIDGET,
+};
+
+const state: Partial<DashboardState> = {
+  dashboardConfiguration: {
+    widgets: [widget],
+    viewport: { duration: '5m' },
```

Comment:
I wish we could make this a default, I feel like I see it everywhere and it's never really necessary to make explicit

Reply:
(No need to do anything to respond to this, just an observation)